semver/range_format.ts

```diff
@@ -1,14 +1,13 @@
 // Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
-import type { SemVerRange } from "./types.ts";
-import { comparatorFormat } from "./comparator_format.ts";
+
+import { formatRange } from "./format_range.ts";
 
 /**
  * Formats the range into a string
  * @example >=0.0.0 || <1.0.0
  * @param range The range to format
  * @returns A string representation of the range
+ *
+ * @deprecated (will be removed after 0.213.0) Use {@linkcode formatRange} instead.
  */
-export function rangeFormat(range: SemVerRange): string {
-  return range.ranges.map((c) => c.map((c) => comparatorFormat(c)).join(" "))
-    .join("||");
-}
+export const rangeFormat = formatRange;
```

Comment:
Please use the below structure to make sure deprecation note appear in IDE:

```ts
/**
 * Formats the range into a string
 * @example >=0.0.0 || <1.0.0
 * @param range The range to format
 * @returns A string representation of the range
 *
 * @deprecated (will be removed after 0.213.0) Use {@linkcode formatRange} instead.
 */
export function rangeFormat(range: SemVerRange): string {
  return formatRange(range);
}
```

Reply:
done